setup.py

```diff
@@ -1,18 +1,11 @@
-import os
 from setuptools import setup, find_packages
 
 
-about = {}
-here = os.path.abspath(os.path.dirname(__file__))
```

Comment:
The use of `__file__` here was problematic due to some weird code calling `setup.py` with `exec`, and given the purpose for which it was being used here, this seems like the cleaner fix to me.